main.nf

```diff
@@ -296,7 +364,7 @@ process idfilter {
      file id_file from id_files_idx_feat_perc_fdr
```

Comment:
would then require to mix this:
`
id_files_idx_feat_perc.mix(id_files_idx_feat_perc_fdr)
`